addons/event/report/event_event_templates.xml

```diff
@@ -188,22 +188,24 @@
         </main>
     </div>
 
+    <div class="oe_structure"></div>
     <div class="row footer o_event_full_page_ticket_footer d-block">
         <div class="o_event_full_page_ticket_powered_by bg-odoo text-white text-center p-2 w-100">
-            <t t-if="event.organizer_id">
-                <span class="fw-bold" t-field="event.organizer_id.name"></span>
+            <span t-if="event.organizer_id">
+                <span class="fw-bold" t-field="event.organizer_id.name">Marc Demo</span>
                 <span t-if="event.organizer_id.phone" class="ps-3 fa fa-phone"/>
-                <span t-if="event.organizer_id.phone" t-field="event.organizer_id.phone"/>
+                <span t-if="event.organizer_id.phone" t-field="event.organizer_id.phone">+123456789</span>
                 <span t-if="event.organizer_id.email_normalized" class="ps-3 fa fa-envelope"/>
-                <span t-if="event.organizer_id.email_normalized" t-field="event.organizer_id.email_normalized"/>
+                <span t-if="event.organizer_id.email_normalized" t-field="event.organizer_id.email_normalized">organizer@email.com</span>
                 <span t-if="event.organizer_id.website" class="ps-3 fa fa-globe"/>
-                <span t-if="event.organizer_id.website" t-field="event.organizer_id.website"/>
-            </t>
+                <span t-if="event.organizer_id.website" t-field="event.organizer_id.website">https://www.organizer.com</span>
```

Comment:
```suggestion
                <span t-if="event.organizer_id.website" t-field="event.organizer_id.website">https://www.example.com</span>
```

You should always favour example.com or example.org as example domains as it is 'officially' set up for that kind of use
https://www.iana.org/help/example-domains